src/Config/Auth.php

```diff
@@ -325,4 +325,30 @@ class Auth extends BaseConfig
      * @var class-string<\Sparks\Shield\Interfaces\UserProvider>
      */
     public $userProvider = 'Sparks\Shield\Models\UserModel';
+
+    /**
+     * Returns the URL that a user should be redirected
+     * to after a successful login.
+     */
+    public static function loginRedirect(): string
+    {
+        $url = setting('Auth.redirects')['login'];
```

Comment:
It feels very strange to call `setting()` when we already have the property instance. I suppose that's the only way we can allow Settings overrides in this case, unless anyone can think of a different approach? Maybe the Settings library needs an "auto-Registrar" option?

Reply:
In this PR, there is no other way to use the values in the database.

It seems Settings needs to get values from the database and set them when a Config class is instantiates.
Is it the "auto-Registrar" option?

Reply:
My "auto-Registrar" idea is a way to have Settings participate in the Registrar stage of Config initialization, so instead of asking for the "settings-ified" value via `settings('Foo.bar')` devs would have the option of having those values already included in the Config file: `config('Foo')->bar`. This would also let Config files use their own properties without having to reroute